src/collect-using.rst

```diff
@@ -7,12 +7,25 @@ Using ODK Collect
 
 ODK Collect is used by :term:`enumerators <enumerator>` to collect survey data by filling out :doc:`forms <form-design-intro>` with :term:`participants <participant>`.
 
-To use ODK Collect:
+Collect is used to:
 
 1. :ref:`Get blank forms <loading-forms-into-collect>` from :ref:`a server or Google Drive <in-app-get-blank-forms>`
 2. :doc:`Fill out surveys  <collect-filling-forms>` with participants
 3. :ref:`Upload completed surveys <uploading-forms>` to a server or Google Drive
 
+You can configure Collect and train your enumerators in different ways to best meet your data collection goals. Use the scenarios below to identify features you might be interested in.
+
+I want to configure a lot of devices the same way
```

Comment:
Did you want this to be a bullet list or have links? It renders just as a paragraph of plain text.

Reply:
Whoops. It was a concept I was playing with. I want to come back to it in another PR. Removed for now.